pkg/project/util/util.go

```diff
@@ -27,6 +29,16 @@ func ConvertNamespace(namespace *kapi.Namespace) *projectapi.Project {
 	}
 }
 
+// ConvertNamespace transforms a Namespace into a Project
+func ConvertNamespaceExternalOrDie(namespace *corev1.Namespace) *projectapi.Project {
+	internalNs := &kapi.Namespace{}
+	if err := kapiv1.Convert_v1_Namespace_To_core_Namespace(namespace, internalNs, nil); err != nil {
+		panic(err)
+	}
+
+	return ConvertNamespace(internalNs)
```

Comment:
don't do this.  rewrite the method for externals.  

Does anyone use the old one now?

Reply:
Thanks, removed call to other method.

@deads2k 
> Does anyone use the old one now?

It is still used by:
- https://github.com/openshift/origin/blob/master/pkg/project/apiserver/registry/project/proxy/proxy.go#L140
- https://github.com/openshift/origin/blob/master/pkg/project/apiserver/registry/project/proxy/proxy.go#L164
- https://github.com/openshift/origin/blob/master/pkg/project/apiserver/registry/project/proxy/proxy.go#L198
- https://github.com/openshift/origin/blob/master/pkg/project/auth/watch.go#L135
- https://github.com/openshift/origin/blob/master/pkg/project/auth/watch.go#L152